src/ngraph/CMakeLists.txt

```diff
@@ -663,16 +663,15 @@ endif()
 #-----------------------------------------------------------------------------------------------
 
 # nGraph
-install(FILES ${CMAKE_BINARY_DIR}/VERSION DESTINATION ${CMAKE_INSTALL_PREFIX})
-install(TARGETS ngraph DESTINATION ${NGRAPH_INSTALL_LIB})  # libngraph.so
+install(TARGETS ngraph DESTINATION ${CMAKE_INSTALL_LIBDIR} EXPORT ngraphTargets)  # libngraph.so
```

Comment:
ngraph install library to `CMAKE_INSTALL_LIBDIR` which is `lib`, `lib64`, etc depending on platform. See https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html

OpenVINO expects the following structure: 
```bash
├── include
│   └── inference_engine.hpp
├── lib
│   └── intel64
│       ├── libcpu_extension_avx2.so
│       ├── libcpu_extension_avx512.so
│       ├── libcpu_extension_sse4.so
│       ├── libinference_engine.so
│       ├── libMKLDNNPlugin.so
│       └── plugins.xml
└── share
    ├── ie_parallel.cmake
    ├── InferenceEngineConfig.cmake
    └── InferenceEngineConfig-version.cmake
```

It means the exported library will have improper location. I think we need to make this destination to be configurable and set from Inference Engine a different location.

Reply:
From my point of view it is good that nGraph follows agreements of each platform and uses default paths.

Reply:
I don't agree that it won't work with OpenVINO because currently you show the structure of Inference Engine component. But nGraph is included to OpenVINO as a separate component and doesn't depend on Inference Engine current structure.

Reply:
Discussed f2f